packages/grid/x-data-grid/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-data-grid",
-  "version": "6.20.1",
+  "version": "6.20.2",
```

Comment:
Was it lerna that suggested bumping `data-grid` packages? 🤔 
It doesn't seem like data grid packages should be released, based on the changes. 🤔

Reply:
Yes, running `yarn release:version` updated all packages versions. I imagined it was the correct flow. Should I downgrade all the packages that do not contain changes ?

Reply:
It seems as if there is some bug in lerna... 🤷 
It compares the with `v6.20.0` instead of `v6.20.1`...
<img width="797" alt="Screenshot 2024-06-20 at 16 48 15" src="https://github.com/mui/mui-x/assets/4941090/5d6278ee-7e42-4304-afcf-896bba9222b0">
I tried doing `git fetch upstream --tags`, but even after fetching the latest tags locally it's still the same... 🤷 
I would probably manually remove these packages from changes.